### ISSUE TITLE
enable sort by injection order

### ISSUE THREAD

**Issue by chubukov** [ElucidataInc/ElMaven#1337]
Sorting by the injection order column (or any other column for that matter) doesn't do anything (in 0.12.0 and other recent versions). 

I think it was disabled after this bug #937 and there were plans to re-enable it. Looks like #1137 was supposed to do that, but I can't recall it ever actually working...

**Comment by saifulbkhan** [comment]
@chubukov You are right. It seems that the only attribute you can sort by right now is the "set" name.
